email_verification/__init__.py

```diff
@@ -35,27 +34,22 @@
 EMAIL_VERIFICATION_SUBJECT = getattr(settings, 'EMAIL_VERIFICATION_SUBJECT',
                                      'Please verify this email address')
 EMAIL_VERIFICATION_TEMPLATE = getattr(settings, 'EMAIL_VERIFICATION_TEMPLATE',
-                                      'email_verification/email.txt')
+                                      'emails/verify_email')
 
 
 # Functions ###################################################################
 
-def send_email_verification(email, request):
+def send_email_verification(email: str):
```

Comment:
@Agrendalath Sorry yeah, I refactored this earlier to take the email address directly instead of the model but I guess I undid that. I now remember why I did so. Adding a typehint here breaks things. It imports a model before Django is ready, so I will just remove the hint, or add it in documentation.

Reply:
> Also, should we move it to `registration/utils.py` to group all methods for sending these emails in one module?

So I think it's best to keep it where it is since it's closer to email verification then registration, and can happen outside the context of registraiton.

Reply:
> So I think it's best to keep it where it is since it's closer to email verification then registration, and can happen outside the context of registraiton.

Sure, sounds reasonable.